setup.py

```diff
@@ -12,13 +12,13 @@
 from setuptools import find_packages, setup, Command
 
 # Package meta-data.
-NAME = 'ethik'
-DESCRIPTION = 'A toolbox for fair and explainable machine learning'
-LONG_DESCRIPTION_CONTENT_TYPE = 'text/markdown'
-URL = 'https://github.com/MaxHalford/ethik'
-EMAIL = 'maxhalford25@gmail.com'
-AUTHOR = 'Max Halford'
-REQUIRES_PYTHON = '>=3.6.0'
+NAME = "ethik"
+DESCRIPTION = "A toolbox for fair and explainable machine learning"
+LONG_DESCRIPTION_CONTENT_TYPE = "text/markdown"
+URL = "https://github.com/MaxHalford/ethik"
+EMAIL = "maxhalford25@gmail.com"
```

Comment:
You can add yourself to the list of authors, see [this example](https://stackoverflow.com/questions/9999829/how-to-specify-multiple-authors-emails-in-setup-py) :)